src/main/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticator.java

```diff
@@ -67,42 +71,47 @@ private AuthUser buildAuthUserFromHeaders(Map<String, String> headers) {
    * Attempt to get the registration status of the current user and set the value on AuthUser
    *
    * @param authUser The AuthUser
-   * @return A cloned AuthUser with Sam registration status
+   * @return A cloned AuthUser with Sam registration status and a Consent User
    */
   private AuthUser getUserWithStatusInfo(AuthUser authUser) {
     if (authUser == null || authUser.getEmail() == null) {
       logWarn("AuthUser/email is null, cannot get user status info");
       return null;
     }
+    try {
+      authUser.setUser(userService.findUserByEmail(authUser.getEmail()));
+    } catch (Exception e) {
+      logException("Error finding Consent user: " + authUser.getEmail(), e);
```

Comment:
This will occur the first time a user logs into duos? So is it an exception or an expected code path?

If it's expected, I think we'd use `logInfo()` here instead of `logException()`, which uses ERROR logging.

Reply:
I'll revisit this block. I think, in this situation, I prefer `logWarn()`. `logException()` triggers a sentry alert which is probably more alerting than we want.